### PR TITLE
Add Travis CI + Docker fix

### PR DESCRIPTION
**Pull request by xvilo**
- Add's a `travis.yml` file, it boots up a container with `openjdk8` installs a known good version of BUCK. (Tried building from source but it ran out of resources) and then runs the tests so we can easily see in MR's if build are failing yes/no

- Fix docker image with a pre-build image that is usable

### CODE REVIEW

**Comment by nicks** [comment]
thanks!

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,7 @@
-FROM nicks/plovr-deps
+FROM xvilo/plovr-base
```
Comment (nicks) [inline Dockerfile]: wait, where does this new docker image come from? how do we build it?

Comment (xvilo) [reply Dockerfile]: Hi @nicks, 

Sorry for the late response, totally forgot about it. It's the `Dockerfile.base` with some extra personal additions. I have reverted this and update both Docker files a bit too! 

Let me know 👍

**Comment by xvilo** [comment]
@nicks could you take another look?

**Review by nicks — APPROVED** [review]

**Comment by nicks** [comment]
thanks!

**Comment by xvilo** [comment]
@nicks Thanks for the approve + merge, however, it seems Travis has not been picked up on master? Should that be enabled first?

**Comment by xvilo** [comment]
For some reason, it also doesn't start on my Fork and on Master. I have checked my Travis CI page, and it didn't show up in the list. However, when I try to go the URL directly I can see a run for the merged branch: https://travis-ci.org/github/xvilo/plovr

Do you happen to have a bit more experience with Travis?

**Pull request merged**